Directory.Build.props

```diff
@@ -5,5 +5,6 @@
       <!-- https://github.com/dotnet/msbuild/issues/2661 -->
       <AddSyntheticProjectReferencesForSolutionDependencies>false</AddSyntheticProjectReferencesForSolutionDependencies>
       <MSBuildEnableWorkloadResolver>false</MSBuildEnableWorkloadResolver>
+      <RunApiCompat>False</RunApiCompat>
```

Comment:
This PR currently disables ApiCompat. This is because even though git handles merges with file moves pretty well, it does __not__ handle changes to `ApiCompatBaseline.txt` well, and these files are going to be the main cause of merge conflicts after this change.

Should we leave this disabled until 11.0 (because breaking changes are expected at this point)?

Reply:
Well, the only issue I can think is its harder to make a breaking change summary for the release.